<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>691</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mininux Replay Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="setPaths_Button">
        <property name="text">
         <string>Set Paths</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="useReplay_Button">
        <property name="text">
         <string>Use selected Replay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="renameReplay_Button">
        <property name="text">
         <string>Rename selected Replay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteReplay_Button">
        <property name="text">
         <string>Delete selected Replay</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="userFolderList_Header">
        <property name="text">
         <string>User data folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="replayDataList_Header">
        <property name="text">
         <string>Replays:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListView" name="replayData_ListView"/>
      </item>
      <item row="2" column="2">
       <widget class="QListView" name="userData_ListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="instructions_Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Instructions:&lt;/p&gt;&lt;p&gt;Click on &amp;quot;Set Paths&amp;quot; and set the paths according to your P+ installation.&lt;/p&gt;&lt;p&gt;Chose the replay you want to use, and hit &amp;quot;Use selected Replay&amp;quot;. You can now open P+ in offline mode and find your replay in Vault &amp;gt; replays. You can also delete replays, or rename them.&lt;/p&gt;&lt;p&gt;If you have any issues, please report them here : &lt;a href=&quot;https://github.com/MininuxDev/mininux-replay-manager&quot;&gt;https://github.com/MininuxDev/mininux-replay-manager&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lastUsedReplay_Label">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>(Last used replay : None)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>setPaths_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setPaths()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useReplay_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>useReplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameReplay_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>renameReplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteReplay_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteReplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setPaths()</slot>
  <slot>useReplay()</slot>
  <slot>renameReplay()</slot>
  <slot>deleteReplay()</slot>
 </slots>
</ui>
